src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2645,12 +2936,8 @@ void processContextElement
   //
   // FIXME P6: Once we allow for ServicePath to be modified, this loop must be looked at.
   //
-  int docs = 0;
-
-  // Used to accumulate error response information, checked at the end
-  bool         attributeAlreadyExistsError = false;
-  std::string  attributeAlreadyExistsList  = "[ ";
 
+  std::vector<BSONObj> results;
   while (moreSafe(cursor))
```

Comment:
Sounds good to me. Close 0.26.0 asap and put this info in an issue, possibly for 0.27.0